client/oci_env/main.py

```diff
@@ -154,6 +156,12 @@ def parse_pulp_cli_command(subparsers):
     parser.set_defaults(func=pulp)
 
 
+def parse_phelper_commands(subparsers):
+    for action in ("start", "stop", "restart"):
```

Comment:
```suggestion
    for action in ("start", "stop", "restart", "status", "clean", "dbreset", "help"):
```

We need to register the rest of the commands:
```
oci-env phelp
```
```
usage: oci-env [-h] [-v] [-e ENV_FILE]
               {compose,exec,db,shell,test,generate-client,pulpcore-manager,profile,poll,pulp,pstart,pstop,prestart}
```

Reply:
Do I really need to? clean and dbreset are already covered in other commands, status doesn't do anything extra than what pulp status does, and help is redundant.  Those extra commands are helpful inside the container, but outside I don't think so.

Reply:
Well, `phelp` can be useful for showing what commends are available for the user. If we wanted to completely mirror the old behaviour, it would be nice to have this change included.

Reply:
Ok, I just added the rest of the phelper commands to the list. I guess it isn't much of a big deal.